pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[tool.ruff]
+line-length = 120
```

Comment:
I can't remember, do we as dimagi use 120 chars as the line length?

Reply:
For flake8, it's currently set to 115 lines for CommCare HQ ([code reference](https://github.com/dimagi/commcare-hq/blob/4274186f9cb6b3e337fe104c8e2019ec13a0f29d/.flake8#L2))

Reply:
We use 115 (to make code review easier (if your browser is maximized on a 1280-wide screen)).

(Personally, I prefer PEP-8 line lengths (79 for code, 72 for docstrings) because (1) it makes code readable when you split your windows vertically -- code on the left, tests on the right -- and (2) long lines of text are ugly and hard to read, which is why newspapers have columns.)